src/core/connection.c

```diff
@@ -6820,6 +6820,31 @@ QuicConnParamGet(
     uint8_t Type;
 
     switch (Param) {
+    
+    case QUIC_PARAM_CONN_ORIG_DEST_CID:
```

Comment:
Let's add this at the end of all of the other parameters' switch statements (not beginning).

Reply:
For sure!